.github/dependabot.yml

```diff
@@ -20,49 +20,10 @@ updates:
     directory: "/"
     schedule:
       interval: daily
```

Comment:
Do you need `package-ecosystem: npm` ?

Reply:
Also missing `labels` but not that big of a deal

Reply:
```suggestion
      interval: daily
      labels:
        - dependabot
```

Reply:
package-ecosystem: npm is there just not in view of diff

Reply:
labels was there too actually just way at the bottom